WordPress/Classes/Networking/PlansFeaturesRemote.swift

```diff
@@ -109,8 +109,9 @@ class PlanFeaturesRemote: ServiceRemoteREST {
         return cacheDirectory.URLByAppendingPathComponent(cacheFilename)
     }
 
-    private func cacheResponseData(responseData: NSData?) {
-        guard let responseData = responseData else { return }
+    private func cacheResponseObject(responseObject: AnyObject) {
+        let data = try? NSJSONSerialization.dataWithJSONObject(responseObject, options: NSJSONWritingOptions())
```

Comment:
I don't like that we're re-encoding the response object into JSON, but I think the problem is that we shouldn't be caching at this layer, or cache the mapped objects instead. The proper fix would be fixing the caching headers in the API and let NSURLSession worry about caching, so I'll let this pass and will look into that instead.